S/Sundials@5/build_tarballs.jl

```diff
@@ -16,7 +16,7 @@ cd $WORKSPACE/srcdir/sundials*
 
 # Set up CFLAGS
 if [[ "${target}" == *-mingw* ]]; then
-    atomic_patch -p1 $WORKSPACE/srcdir/patches/Sundials_windows.patch
+    # atomic_patch -p1 $WORKSPACE/srcdir/patches/Sundials_windows.patch
     # Work around https://github.com/LLNL/sundials/issues/29
     export CFLAGS="${CFLAGS} -DBUILD_SUNDIALS_LIBRARY"
```

Comment:
Also this `export` shouldn't be necessary, according to https://github.com/LLNL/sundials/issues/29#issuecomment-606679985.  It's probably useful to see whether that's the case

Reply:
Agreed, I just wanted to go one change at a time in case it breaks.

Reply:
The windows builds are failing so the patch probably is needed. I thought the upstream issue referred to not needing the -DBUILD_SUNDIALS_LIBRARY. I don't think we filed an issue for the patch we are carrying.

Reply:
Trying again

Reply:
I guess it did not work.

Reply:
Now it does. We can merge so I can use this in Sundials.jl